### PR TITLE
Update tutorial to use oam-local

### PR DESCRIPTION
**Pull request by rynowak**
Simplifies the tutorial dramatically because:
- the user doesn't need multiple terminals
- now all of the coding/deployment happens from your local machine (no VMs)
- works with normal permissions levels

Worthing doing a read through and checking that I got all of the
references to a VM and minikube - I think I cleaned it all up.

### CODE REVIEW

**Comment by rynowak** [comment]
Right now this PR builds on the changes from https://github.com/Azure/azure-oam-solution/pull/13 since it's edits to the tutorial. I'll update when that is merged.

**IMPORTANT NOTE**

If you test this out right now the instructions won't quite work because we have to refer to URLs that are branch specific 😆 

You can run the ARM template *before* this is merged by doing:

```sh
az deployment group create \
  --template-file template.json \
  --resource-group rynowak-testing \
  --parameter "sshRSAPublicKey=$(<~/.ssh/id_rsa.pub)" \
  --parameter "servicePrincipalClientId=$(<creds.json | jq '.clientId' --raw-output)" \
  --parameter "servicePrincipalClientSecret=$(<creds.json | jq '.clientSecret' --raw-output)" \
  --parameter "_scriptUri=https://raw.githubusercontent.com/azure/azure-oam-solution/rynowak/local/scripts/install-oam-local.sh"
```

**Comment by artursouza** [comment]
Lets' merge the other one before reviewing this to make the diff more readable.

**Comment by rynowak** [comment]
Rebased and updated

File: README.md [hunk 1]
```diff
@@ -102,21 +125,17 @@ Make a note of the resource group name you choose for reference later.
...
+Run the following command to deploy a Kubernetes cluster with support for OAM.
 
-**Bash/Zsh**:
 ```sh
 az deployment group create \
-  --template-uri https://raw.githubusercontent.com/Azure/azure-oam-solution/master/template.json \
-  --resource-group $OAM_TUTORIAL_RESOURCE_GROUP_NAME  \
-  --parameter "adminPasswordOrKey=$(<~/.ssh/id_rsa.pub)" \
-  --parameter "servicePrincipal=$(<creds.json)"
+  --template-file template.json \
```
Comment (artursouza) [inline README.md:132]: template-uri

File: README.md [hunk 2]
```diff
@@ -102,21 +125,17 @@ Make a note of the resource group name you choose for reference later.
...
+  --resource-group rynowak-testing \
```
Comment (artursouza) [inline README.md:133]: use the env name for RG

**Comment by rynowak** [comment]
Updated

**Pull request merged**